platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -131,7 +170,11 @@ public void setUp() throws Exception {
   public void testInit() throws Exception {
     ApplicationServiceBean serviceBean =
         new ApplicationServiceBean(
-            testAppService, testConfigAdminExt, mBeanServer, mockFeaturesService);
+            testAppService,
```

Comment:
❓Instead of repeating these lines everywhere, should we create a `private` factory method that creates the `ApplicationServiceBean` with all the right arguments?

Reply:
trying to limit the scope of this ticket.